src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  position: relative;
+  height: 60px;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
```

Comment:
Do you really need to make it absolute? Consider just setting correct padding to header or margins for logo